waimak-core/src/main/scala/com/coxautodata/waimak/dataflow/DataFlowAction.scala

```diff
@@ -31,16 +31,28 @@ trait DataFlowAction[C] {
     */
   val outputLabels: List[String]
 
+  /**
+    * Unique id of the action, but using it for adding behaviours can be problematic due to Interceptors that are
+    * defined at a much later stage. Because of that ActionSchedulers must use this guid.
```

Comment:
should this say "ActionSchedulers must **not** use this guid."

Reply:
yep, fixed